Maple2.Server.Game/PacketHandlers/ItemUseHandler.cs

```diff
@@ -18,5 +19,23 @@ public class ItemUseHandler : PacketHandler<GameSession> {
             Logger.Warning("RequestItemUse for invalid item:{ItemUid}", itemUid);
             return;
         }
+
+        switch (item.Metadata.Function?.Name)
+        {
+            case "StoryBook":
+                HandleStoryBook(session, item);
+                break;
+            default:
+                Logger.Warning("Unhandled item function: {item.Metadata.Function?.Name}", item.Metadata.Function?.Name);
```

Comment:
Just "Name" as field is fine (instead of "item.Metadata.Function?.Name"), also might want to switch this to use `Type` as well.